modules/tinymce/src/core/main/ts/focus/FocusController.ts

```diff
@@ -163,8 +144,6 @@ const setup = (editorManager: EditorManager): void => {
 };
 
 export {
-  setup,
-  isEditorUIElement,
-  isEditorContentAreaElement,
-  isUIElement
+  isEditorContentAreaElement, isEditorUIElement, isUIElement, setup
```

Comment:
Undo this change. The TS LSP sometimes messes this line up if you run organize imports.